manifests/repos.pp

```diff
@@ -1,5 +1,5 @@
 # Set up a repository for foreman
-define foreman::install::repos(
```

Comment:
This is going to cause a problem in foreman_proxy, as it's calling this class directly: https://github.com/theforeman/puppet-foreman_proxy/blob/2.5.0/manifests/install.pp#L4-L7

Should we avoid renaming it?